src/EventHub/EventHub/help/Az.EventHub.md

```diff
@@ -60,25 +58,25 @@ Gets lists of resources that supports Privatelinks.
 Gets the details of an EventHub schema group.
 
 ### [New-AzEventHub](New-AzEventHub.md)
-Create a new Event Hub as a nested resource within a Namespace.
+create a new Event Hub as a nested resource within a Namespace.
 
 ### [New-AzEventHubApplicationGroup](New-AzEventHubApplicationGroup.md)
-Create an ApplicationGroup for a Namespace.
+create an ApplicationGroup for a Namespace.
 
 ### [New-AzEventHubAuthorizationRule](New-AzEventHubAuthorizationRule.md)
 Creates an EventHub Authorization Rule
 
 ### [New-AzEventHubAuthorizationRuleSASToken](New-AzEventHubAuthorizationRuleSASToken.md)
-Generates a SAS token for Azure eventhub authorization rule of namespace/eventhub. 
+Generates a SAS token for Azure eventhub authorization rule of namespace/eventhub.
 
 ### [New-AzEventHubCluster](New-AzEventHubCluster.md)
-Create an instance of an Event Hubs Cluster.
+create an instance of an Event Hubs Cluster.
 
 ### [New-AzEventHubConsumerGroup](New-AzEventHubConsumerGroup.md)
-Create an Event Hubs consumer group as a nested resource within a Namespace.
+create an Event Hubs consumer group as a nested resource within a Namespace.
```

Comment:
The first letter c is not capitalized.

Reply:
Update to this PR forthcoming, but incorrect in [the Synopsis of the reference doc](https://github.com/Azure/azure-powershell/blob/main/src/EventHub/EventHub/help/New-AzEventHubConsumerGroup.md#synopsis).